src/game/mod.rs

```diff
@@ -27,11 +29,12 @@ impl Game {
             score: 0,
             glyph_cache: GlyphCache::new("./assets/FiraSans-Regular.ttf").unwrap(),
             window_size: window_size,
+            asteroid_timer: 1.0,
+            asteroid_timer_max: 10.0,
```

Comment:
Overall I think we should give the asteroids a pretty big bump in spawn rate. In the beginning you wait a while before another asteroid spawns.

I really like how the difficulty starts to ramp up. When they start really coming in, it gets very challenging. Once we do #7/Random radii, it will be very interesting.

I would suggest starting the difficulty a bit higher (so as not to lose interest) and have the slope of the increase of the difficulty be just a little less steep.